internal/ui/playlists.go

```diff
@@ -4,29 +4,21 @@ import (
 	"github.com/charmbracelet/bubbles/list"
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/ehabshaaban/lazyfy/api"
-	"github.com/zmb3/spotify/v2"
 )
 
-var (
-	playlistItems []list.Item
-	state         bool = false
-	foo           TrackModel
-	trackModels   map[spotify.ID]TrackModel = make(map[spotify.ID]TrackModel)
-)
+var trackModels = make(map[string]*trackModel)
 
-type playlistItem struct {
-	id, name string
-}
+type playlist api.Playlist
 
-func (p playlistItem) Title() string { return p.name }
+func (p playlist) Title() string { return p.Name }
```

Comment:
I'm not super sure, but most probably if you're using `playlist/api.Playlist` as a pointer, then it would make since to also declare struct functions as pointer too like:

```
func (p *playlist) Title() string { return p.Name }
```